lib/teslamate_web/live/car_live/summary.html.heex

```diff
@@ -115,6 +115,14 @@
             <span class="mdi mdi-shield-check"></span>
           </span>
         <% end %>
+        <%= if @summary.center_display_state == 7 do %>
```

Comment:
This is not releated to your PR. Please opene separate PR for this

Reply:
Additionally this is included since ages via: https://github.com/teslamate-org/teslamate/blob/63b76779e6d2b59e7e839e550017b8004e9acf29/lib/teslamate_web/live/car_live/summary.html.heex#L110

Reply:
It's when Sentry Mode is active. Not when it's triggered and recording.

Reply:
> This is not releated to your PR. Please opene separate PR for this

Sorry, my bad... #facepalm

- https://github.com/teslamate-org/teslamate/pull/4042
- https://github.com/teslamate-org/teslamate/pull/4043